### ISSUE TITLE
NXThemes Installer ver 2.7 crashes when "Input text" is selected

### ISSUE THREAD

**Issue by justincys** [exelix11/SwitchThemeInjector#115]
In non-applet mode, if I navigate to Download themes and press A on Input text, the app hangs on me and I have to restart the Switch. Also happens from time to time after I install a theme. Have uninstalled all themes since but would really love to be able to use themes again. Only happened after the folders update. I have recreated my themes with the new v4.7 program but still getting crashes at times.

Not sure what could be wrong. if it matters, the other sys-modules enabled are: sys-tweak, sys-clk, saltynx, ldn_mitm, sys-hidplus, and sys-ftpd-light. Have changed a few game icons and a game's name via NX Game Icon Customizer too.

**Comment by exelix11** [comment]
That's odd, the text field should invoke the keyboard applet not sure why it freezes.
From the top of my mind:
1) Do you use title takeover (R while launching a game) or an nsp build of the theme installer ? Note that nsp builds are not provided by me and not supported.
2) Does the software keyboard work in other games ?
3) Can you try if the same happens without sysmoduels running in the background ?

**Comment by justincys** [comment]
1. Yes I used title takeover.
2. I didn't test for other games.
3. I'm now trying various things like reinstalling a few sys modules. I noticed my sys-ftpd-light suddenly changed to sys-ftpd-10k and am not sure how that happened. It could really just be a big ol' user issue.

Update: Have tried reinstalling sys-tweak and things seem to be going well now. I can invoke the keyboard. Sorry to make a thread for this, but it really crashed on me every time I opened the keyboard inside NX Themes Installer so I was convinced this app was the issue.
